docs/USING.rst

```diff
@@ -6,10 +6,22 @@ API are documented in `apidesc.md <apidesc.md>`_, and the mapping to HTTP is
 described in `rest_api.md <rest_api.md>`_.
 
 The ``haas`` command line tool is a wrapper around this API. Running ``haas
-help`` will display an overview of the available commands. Just as with ``haas
-serve_networks``, ``haas.cfg`` must be in your working directory. But here,
-only the ``client`` section is needed, so users without access to the global
-configuration file may use a more minimal one.
+help`` will display an overview of the available commands. To tell ``haas``  which HaaS instance to use, be sure to do one of:
+
+1. Set the ``HAAS_ENDPOINT`` environmental variable. An example (compatible
+   with the default development configuration) would be ``http://127.0.0.1:5000``
```

Comment:
There's nothing that's really stated as being the "default"; If you want to reference one of the example configs I would explicitly mention the file name.